source/includes/_authenticated_api_attendees.md

```diff
@@ -0,0 +1,126 @@
+## Attendees
+
+Attendees are actions that people have taken on Events. All attendee actions are namespaced by URL underneath the eventss that the attendee was recorded against.
```

Comment:
Hmm, I feel like it might be more illuminating to say "Attendees are member RSVPs to Events" or something -- while it's true that these are actions, it doesn't tell the user much about what they represent in the real world.